src/css/main.css

```diff
@@ -39,12 +39,16 @@
 .hero-txt > .btn-primary {
     margin-top: 1rem;
     background-color: var(--accent-color);
-    color: var(--dark-black);
+    color: #efefef;
     font-weight: 400;
```

Comment:
Prefer using color variables

Reply:
Thank you for the detailed review Jowel, will work on all the suggestions for sure!